ansible/roles/collectory/vars/main.yml

```diff
@@ -13,4 +13,6 @@ packaging: "war"
 
 collectory_war_url: "{{ collectory_maven_repo_ws_url | default(maven_repo_ws_url) }}"
 
-collectory_data_dir: "{{ data_dir }}/{{ collectory_app | default('generic-collectory') }}"
\ No newline at end of file
+collectory_data_dir: "{{ data_dir }}/{{ collectory_app | default('generic-collectory') }}"
```

Comment:
generic-collectory is not something we support any more. Suggest just replacing with 'collectory'